src/NuGet.Services.EndToEnd/NuGet.Services.EndToEnd.csproj

```diff
@@ -157,11 +157,14 @@
     <PackageReference Include="Microsoft.Extensions.Configuration.Binder">
       <Version>2.2.0</Version>
     </PackageReference>
+    <PackageReference Include="Newtonsoft.Json">
+      <Version>13.0.1</Version>
+    </PackageReference>
```

Comment:
since you are adding a top level dependency, doesn't this need binding redirect for transitive dependencies of Newtosoft.Json?

Reply:
I think we only need explicit binding redirects only for web projects (that use `web.config` instead of `app.config`).

Reply:
That's correct.